### ISSUE TITLE
Canon M50 Compatibility

### ISSUE THREAD

**Issue by shadimar69** [iebyt/cbremote#2]
Error: Canon M50 not detecting CB Remote APK as a valid Bluetooth Remote.

Symtoms: In 'Pairing mode' CB detects bluetooth of Canon M50 in "Bluetooth Remote" mode, 'links' for less than 1 second, camera throws "Connection target not found"
![whatsapp image 2018-11-02 at 10 58 59 am](https://user-images.githubusercontent.com/44565793/47891551-a7fe8e00-de8e-11e8-8181-6d0755bf9876.jpeg)
 error. CB disconnects. Change mode in M50 over to "Android Phone" mode, CB shows a linked connection, however CB has no functionality at all.

NOTES: Happy to be a test subject for any new build APK's of CB.

**Comment by ruohki** [comment]
It is connecting but not taking pictures on my M50

**Comment by ids1024** [comment]
@ruohki You need to set the camera to self-timer/remote for it to work. At least on the t7i. Your camera's manual should say how to use a Bluetooth remote with it.

**Comment by ruohki** [comment]
@ids1024 Yes found it after websearch. Sadly the Manual of the M50 is super poor. Anyways it does not work very well when the display preview is enabled. But for series of long exposure shots this is just what i needed since the M50 does not have a remote jack.

What i noticed is that as soon as the phone goes black the connection is canceled and it requires unpairing and repairing to get it working again (Huawai P Smart 2019)

**Comment by maxmacstn** [comment]
Hi, I have successfully connected this app with my EOS M50 and it works fine. But every time after turn off the camera, I need to clear Bluetooth connection setting in camera and redo the paring process again. After I do some research, I found that the the camera is not spreading the Bluetooth signal (I tried to search with different devices e.g. nRF Connect app and ESP32 but those couldn't found.) even though in connection info page it said "Connecting..."

I guess the camera may be changed to Client mode and searching for the remote which will be the Server mode, do some handshake and then it can connected again. Or Canon just uses the unstandardized protocol.
